pkgs/os-specific/linux/mcontrolcenter/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, stdenv, qtbase, wrapQtAppsHook, qttools, makeDesktopItem, copyDesktopItems, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mcontrolcenter";
+  version = "0.4.1";
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "MControlCenter";
+      exec = "mcontrolcenter";
+      icon = "mcontrolcenter";
+      comment = finalAttrs.meta.description;
+      desktopName = "MControlCenter";
+      categories = [ "System" ];
+    })
+  ];
+
+  buildInputs = [
+    qtbase
+  ];
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    qttools
+    copyDesktopItems
+    cmake
+  ];
+
+  src = fetchFromGitHub {
+    owner = "dmitry-s93";
+    repo = "MControlCenter";
+    rev = "f04ca9878f0ccfa5ea849a3bd17bcb79a5f52a0a";
+    hash = "sha256-SV78OVRGzy2zFLT3xqeUtbjlh81Z97PVao18P3h/8dI=";
+  };
+
+  patches = [ ./modprobe.patch ];
+
+  cmakeFlags = [
+    "-DENABLE_TESTING=OFF"
```

Comment:
Why are you disabling tests ? Does this project even has test, or include CTest, for `ENABLE_TESTING` to be used for anything ?